src/spikeinterface/postprocessing/template_similarity.py

```diff
@@ -60,25 +83,109 @@ def _get_data(self):
 compute_template_similarity = ComputeTemplateSimilarity.function_factory()
 
 
-def compute_similarity_with_templates_array(templates_array, other_templates_array, method):
+def compute_similarity_with_templates_array(
+    templates_array, other_templates_array, method, support="union", n_shifts=0, sparsity=None, other_sparsity=None
```

Comment:
I would use num_shifts here instead of n_shifts for consistency with other places.